conf/log4j2.xml.template

```diff
@@ -22,24 +22,20 @@
   -->
 <Configuration status="INFO">
     <Appenders>
-        <Console name="stdout" target="SYSTEM_OUT">
-            <!--
-              ~ In the pattern layout configuration below, we specify an explicit `%ex` conversion
-              ~ pattern for logging Throwables. If this was omitted, then (by default) Log4J would
-              ~ implicitly add an `%xEx` conversion pattern which logs stacktraces with additional
-              ~ class packaging information. That extra information can sometimes add a substantial
-              ~ performance overhead, so we disable it in our default logging config.
-              -->
+        <RollingRandomAccessFile name="rollingBySize" fileName="${env:CELEBORN_LOG_DIR}/celeborn.log" filePattern="${env:CELEBORN_LOG_DIR}/celeborn.log.%d" ignoreExceptions="false" append="true">
             <PatternLayout pattern="%d{yy/MM/dd HH:mm:ss,SSS} %p [%t] %c{1}: %m%n%ex"/>
-        </Console>
```

Comment:
please reserve the console, it's very useful when manage celeborn using supervisord or in K8s mode. you can add the file separately.

Reply:
OK, I'll reserve the console and add a comment here so that if a user can choose.

Reply:
Emm... normally we choose `RollingFile`,  I'm not sure what's difference between them.

Reply:
Better log performance.

> The RollingRandomAccessFileAppender is similar to the standard [RollingFileAppender](https://logging.apache.org/log4j/2.x/manual/appenders.html#RollingFileAppender) except it is always buffered (this cannot be switched off) and internally it uses a ByteBuffer + RandomAccessFile instead of a BufferedOutputStream. We saw a 20-200% performance improvement compared to RollingFileAppender with "bufferedIO=true" in our [measurements](https://logging.apache.org/log4j/2.x/performance.html#whichAppender).